README.md

```diff
@@ -1,26 +1,14 @@
-# app
+# Brewhops App [![CircleCI](https://circleci.com/gh/brewhops/app.svg?style=svg)](https://circleci.com/gh/brewhops/app) [![codecov](https://codecov.io/gh/brewhops/app/branch/master/graph/badge.svg)](https://codecov.io/gh/brewhops/app)
 
-## Installs dependencies
-```
-npm i
-```
+The app is written in VueJs and bootstrapped by the vue cli. If you need to add any additional webpack configuration you can extend the exported object in `vue.config.js`. If contributing, please write unit tests for any feature that you add, we're doing ourselves a favor!
 
-### Compiles and hot-reloads for development
-```
-npm run dev
-```
+## Commands
+- `npm ci`: preferred because it will install dependencies based on package-lock.json, enforcing consistent package versions
+- `npm run dev`: starts the dev server and hot-reloads on file changes
+- `npm run build`: builds and minifies the project for production
+- `npm run test`: runs unit tests and displays code coverage
+- `npm run lint`: runs the linter on the project to enforce consistent code styling
 
-### Compiles and minifies for production 
-```
-npm run build
-```
-
-### Runs unit tests
-```
-npm run test
-```
-
-### Lints and fixes files
-```
-npm run lint
-```
\ No newline at end of file
+## Deployment Commands 
+- `npm run deploy` - __Please don't run this!__ it will deploy to production
+- `npm run deploy-staging` - use this if you want to deploy your current changes to a `now` staging environment so you can test them
```

Comment:
It might be nice to have some more explaination about where exactly the now staging environment is, or how it is managed, but that also might be because I haven't even tried doing it yet and there might be some direction there I am missing.

Reply:
I can put that in there, but I can't put too many people on our `now` account or I'll have to pay more. I decided to automate all deployments to staging so whenever you push code, you'll get a deployment that you can access on the brewhops-deployments channel. But for prod deployments, it is kind of designed to be a very manual thing.